languoids/tree/nilo1247/sout2830/tato1241/geme1247/nort3277/bura1273/md.ini

```diff
@@ -15,8 +15,8 @@ multitree =
 	Buradiga
 	Bureadiga
 glottolog = 
-	Buradiiga
 	Buradiga
+	Buradiiga
```

Comment:
Not sure if there is a typo here (multitree : Buraadiiga vs. glottolog Buradiiga)

Reply:
Both are typos, Buraadiga is correct (as per Rottland 1982 etc). I'll fix it for Glottolog

Reply:
Thanks!